package.json

```diff
@@ -24,13 +25,18 @@
   "dependencies": {
     "caporal": "^0.9.0",
     "chalk": "^2.3.0",
+    "cli-spinner": "^0.2.8",
     "figlet": "^1.2.0",
-    "inquirer": "^5.0.0"
+    "inquirer": "^5.0.0",
```

Comment:
The [package-lock.json](https://github.com/AutolabJS/autolabcli/blob/autolabcli-ng/package-lock.json) needs update as well.